README.md

```diff
@@ -6,49 +6,58 @@ Status:
 [![Build Status](https://img.shields.io/travis/CanopyTax/sofe.svg?style=flat-square)](https://travis-ci.org/CanopyTax/sofe)
 [![Code Coverage](https://img.shields.io/codecov/c/github/CanopyTax/sofe.svg?style=flat-square)](https://codecov.io/github/CanopyTax/sofe)
 
-
 ## Motivation for Front-end Services
+1. Deploy once, update everywhere
 1. Easier to organize and scale teams
-2. Promotes culture of ownership, autonomy, responsibility and accountability
-3. Granular release planning
-4. High trust, low processes
-5. Encourages re-use
-6. Avoid the re-write by providing easier upgrade ability. Less framework lock-in.
-7. Deploy once, update everywhere
-
-## Front-end Services
+1. Promotes culture of ownership, autonomy, responsibility and accountability
+1. Avoid the re-write by making fewer architectural decisions that affect *everything*.
+1. Testability
+1. High trust, low processes
+1. Granular release planning
+1. Encourages re-use
+1. Less framework lock-in
+
+## Definition of a Front-end Service
 1. Independently deployable
-2. Bounded context (100% self-contained)
-3. Interaction is strictly via API
-4. Has it's own data store, does not share same data structures as other services. Individually responsible for server communication.
```

Comment:
Interaction should be strictly via API for front-end services as well. They shouldn't be writing to globals, or sharing global data structures with other services.